<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-SF079DSF" failures="1" tests="1" name="TestRunner.FailedTestRunner" time="8.351" errors="0" timestamp="2024-10-02T18:03:51 IST" skipped="0">
  <testcase classname="TestRunner.FailedTestRunner" name="runScenario" time="8.351">
    <failure message="expected [No data] but found [Beetroot]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [No data] but found [Beetroot]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.OfferPageStepDefinition.user_validate_the_search_product_on_offer_page(OfferPageStepDefinition.java:32)
at âœ½.User validate the search product Beet on Offer page(file:///E:/Selenium/SeleniumWorkspace/First/E2ECucumber/src/test/java/features/CucumberFeature.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- TestRunner.FailedTestRunner -->
